pkgs/applications/gis/grass/noLn.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/include/Make/Install.make b/include/Make/Install.make
+index 0aba138..ad6af49 100644
+--- a/include/Make/Install.make
++++ b/include/Make/Install.make
+@@ -118,7 +118,7 @@ real-install: | $(INST_DIR) $(UNIX_BIN)
+ 
+ ifneq ($(findstring darwin,$(ARCH)),)
+ 	@# enable OSX Help Viewer
+-	@/bin/ln -sfh "$(INST_DIR)/docs/html" /Library/Documentation/Help/GRASS-$(GRASS_VERSION_MAJOR).$(GRASS_VERSION_MINOR)
++	#@/bin/ln -sfh "$(INST_DIR)/docs/html" /Library/Documentation/Help/GRASS-$(GRASS_VERSION_MAJOR).$(GRASS_VERSION_MINOR)
```

Comment:
Is this commenting it out? If so, maybe prefer just removing it completely.

Reply:
Yes it is commented out. I can remove it completely as well, I was just trying to make the least change possible.

Reply:
I would just remove it, that makes it much easier to read the diff.